### ISSUE TITLE
Hi-DPI on Windows brings issues

### ISSUE THREAD

**Issue by runxel** [CloudCompare/CloudCompare#1583]
**Describe the bug**

**How to reproduce**
Open CloudCompare on Windows on a second display which has high DPI (i.e. 3840 x 2160).
In the Windows settings this second display has a scaling factor of 150% set (as per recommendation).
This however breaks CloudCompare and smashes the menu band into the title bar, as you can see:

![grafik](https://user-images.githubusercontent.com/2277182/150699826-5f0f4958-1404-4cea-972d-043eab9b57e4.png)

Also it seems that the scale factor does not help with CC – everything is ***super*** small and unusable, since it's easy to misclick when everythin is just a few pixels big.

**Expected behaviour**
CloudCompare should support hi dpi displays and Windows scale factors correctly.
I guess this is to be done with an appmanifest.

**Additional context**

**Your environment**
 - CC Version (or git hash/tag): v2.11.3 (Anoia) 
 - OS & Version: Windows 10 Version 21H1 Build 19043
 - Graphics card: nVidia GeForce GTX 960M
 - Qt Version (if compiling): n/a

**Comment by runxel** [comment]
Just discovered the save dialog is comically small as well.
So it's absolutely connected to not reading the correct scaling for each display, but only from the main display (which happens to be a normal laptop screen in this case here).

**Comment by dgirardeau** [comment]
Yes, screens with different resolutions is a challenge, and neither Qt nor Windows provide a good solution to it (https://doc.qt.io/qt-5/highdpi.html)

Can you try to run CloudCompare with the following command?
`CloudCompare -platform windows:dpiawareness=2`

And see if it helps even a little?

**Comment by runxel** [comment]
@dgirardeau
With that command CloudCompare doesn't even start...

What seems to help is this setting inside the compatibility options:
(This scales the app, which makes everything slightly blurry, but at least now I don't misclick any longer)

<img width="310" alt="cc1" src="https://user-images.githubusercontent.com/2277182/152655092-e36535d9-2039-442d-aafc-4d5316b6a64e.png">

![grafik](https://user-images.githubusercontent.com/2277182/152655158-b369faf1-aac0-4985-8c5d-8aad62816eb7.png)

**Comment by dgirardeau** [comment]
Ah, tanks for the test, and thanks for the hint!

Let's hope that Qt 6 will improve that in the future...
